tests/data/central-activities-zone.sqlite3.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Do we have a use case for the fields other than the hash in this JSON document?

Reply:
Not at the moment but I figure if we want to change tactics and include other things in the future then we can. Python dicts are also to store as json etc.